src/main/java/sdu/se9/tv2/management/system/App.java

```diff
@@ -19,6 +20,7 @@
     public void start(Stage stage) throws IOException {
         scene = new Scene(loadFXML("primary"), 640, 480);
         stage.setScene(scene);
+        stage.setTitle("Management System");
```

Comment:
"Management System" fortæller ikke noget om hvad programmet bruges til, kunne kalde det "TV2 credit system" eller "TV2 credit management system" i stedet for.